services/admin/tsconfig.json

```diff
@@ -0,0 +1,28 @@
+{
+  "extends": "../../tsconfig.shared.json",
+  "include": ["src", "test"],
+  "exclude": [],
+  "compilerOptions": {
+    "allowJs": true,
+    "allowSyntheticDefaultImports": true,
+    "esModuleInterop": true,
+    "forceConsistentCasingInFileNames": true,
+    "isolatedModules": false,
```

Comment:
We're probably going to want this to be `true` [for vite](https://vitejs.dev/guide/features.html#typescript-compiler-options), so let's just do that now.